package.json

```
@@ -22,7 +22,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "node src/scripts/orderData.js && git add src/lib"
+      "pre-commit": "npm run lint && node src/scripts/orderData.js && git add src/lib"
```

Usar yarn ao inves de npm

```suggestion
      "pre-commit": "yarn run lint && node src/scripts/orderData.js && git add src/lib"
```

---

Isso é culpa no meu vicio de npm, my bad.
Troco sim :smile: